src/notebook_unit1.ipynb

```diff
@@ -95,9 +95,9 @@
    "source": [
```

Comment:
<p>This formula is quite confusing due to the time variable not being consistent. Better:</p><p>$\Psi_{corr}(t) = \langle \Psi(0) | \Psi(t) \rangle = \int_0^{t} \Psi(0)^* \Psi(t') dt'$</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pauleonix/ssd_course/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>